package.json

```diff
@@ -17,6 +17,9 @@
   },
   "homepage": "https://github.com/acushlakoncept/restaurant-page#readme",
   "devDependencies": {
+    "eslint": "^6.8.0",
```

Comment:
- [ ] Please add necessary run common commands like build, development and start in package.json inside `scripts'.